scala/core/src/partprops/Parts.scala

```diff
@@ -0,0 +1,141 @@
+package semagrams
+
+import upickle.default._
+  
+case class Part(id:UUID,ob:Ob) extends Entity:
+
+  val ty = ob
+
+  override def toString = if ob.label == ""
```

Comment:
Maybe something that put punctuation between the label and the rand, and also made a clear distinction between anonymous parts, and parts with label "Part"